### ISSUE TITLE
[Feat]: Spotdl as a source

### ISSUE THREAD

**Issue by Mateleo** [RandomNinjaAtk/docker-lidarr-extended#170]
https://github.com/spotDL/spotify-downloader

Spotdl is a fantastic cli for downloading music !

**Comment by KINGLIFER** [comment]
This would be awesome

**Comment by csimpi** [comment]
Not that fast, it's just downloading matching videos from Youtube and converting them to audio files. I got a low-quality music library as a result.

**Comment by RandomNinjaAtk** [comment]
FYI, I won't be working on this... PR welcome

**Comment by KINGLIFER** [comment]
Have you seen audials? It would be awesome to pull from the actual sources not YouTube. 

Thanks for the input.

**Comment by RandomNinjaAtk** [comment]
I don't see a need to integrate any other clients, it's a lot of work and what's here already works just fine

**Comment by csimpi** [comment]
Yeah I have to agree with @RandomNinjaAtk I don't see any reason to make lidarr to something that is not.
BTW what Lidarr needs is a better music recognition flow. Many of songs are not recognized by Lidarr (50% of my library). Shazam API ships crazy amount of data, I am experimenting it now.
